### PR TITLE
MESH-1268/refactor cli tests

### PR DESCRIPTION
**Pull request by JMoore96**
Addressing reusing identities

### CODE REVIEW

**Review by adamdossa — APPROVED** [review]

File: scripts/cli/util/init.js [hunk 1]
```diff
@@ -132,6 +125,21 @@ const generateEntityFromUri = async function (api, uri) {
   return entity;
 };
 
+const generateRandomEntity = async function (api) {
+  let entity = await generateEntityFromUri(api, cryptoRandomString({length: 10}));
+  return entity;
```
Comment (vkomenda) [inline scripts/cli/util/init.js:130]:
```suggestion
  await generateEntityFromUri(api, cryptoRandomString({length: 10}))
```

File: scripts/cli/util/init.js [hunk 2]
```diff
@@ -132,6 +125,21 @@ const generateEntityFromUri = async function (api, uri) {
...
+}
+
+const generateRandomTicker = async function (api) {
+  let ticker = cryptoRandomString({length: 12, type: 'distinguishable'});
+  return ticker;
```
Comment (vkomenda) [inline scripts/cli/util/init.js:135]:
```suggestion
  cryptoRandomString({length: 12, type: 'distinguishable'})
```

File: scripts/cli/util/init.js [hunk 3]
```diff
@@ -132,6 +125,21 @@ const generateEntityFromUri = async function (api, uri) {
...
+}
+
+const generateRandomKey = async function (api) {
+  let ticker = cryptoRandomString({length: 12, type: 'alphanumeric'});
+  return ticker;
```
Comment (vkomenda) [inline scripts/cli/util/init.js:140]:
```suggestion
  cryptoRandomString({length: 12, type: 'alphanumeric'})
```

**Review by vkomenda — APPROVED** [review]
Looks good. A few possible improvements.

**Pull request merged**